app/controllers/items_controller.rb

```diff
@@ -9,28 +9,28 @@ def create
     if item_params[:name].blank?
       @item = Item.new
       flash[:error] = "Invalid"
-      render :home and return
+      render :index and return
```

Comment:
Nice job only changing one big thing at a time. I'm sure it hurts a little to leave this here :)